src-v2/Not/Not.Blazor/TM/Models/MultiSelectListModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace Not.Blazor.TM.Models;
+
+public class MultiSelectListModel
+{
+    public MultiSelectListModel(string value, string description)
+    {
+        Value = value;
+        Description = description;
+    }
+
+    public string Value { get; set; }
+    public string Description { get; set; }
+
+    public static IEnumerable<MultiSelectListModel> MultiSelectList<T>()
+    {   
+        var enumValues = Enum.GetValues(typeof(T)).Cast<Enum>();
+        var selectItems = enumValues.Select(s => new MultiSelectListModel(s.ToString(), EnumHelper.GetDescription(s)));
+        return selectItems;
+    }
+}
+
+public static class EnumHelper
+{
+    public static string GetDescription(this Enum value)
+    {
+        FieldInfo fieldInfo = value.GetType().GetField(value.ToString());
+        DescriptionAttribute descriptionAttribute = fieldInfo.GetCustomAttributes(typeof(DescriptionAttribute), false)
+                                                             .FirstOrDefault() as DescriptionAttribute;
+        return descriptionAttribute == null ? value.ToString() : descriptionAttribute.Description;
```

Comment:
Overall pretty good. I have a few formatting grapes, but more importantly: keep in mind that when we are using reflection we want to be doing that using the `ReflectionHelper` class.